library/swipl

```diff
@@ -1,6 +1,10 @@
 Maintainers: Dave Curylo <dave@curylo.org> (@ninjarobot)
 GitRepo: https://github.com/SWI-Prolog/docker-swipl.git
 
-Tags: latest, 7.5.11
+Tags: latest, 7.5.12
+GitCommit: c62441347c7984269ab315fbf042850cbe3407fa
+Directory: 7.5.12/stretch
+
+Tags: 7.5.11
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)